rialto_airflow/harvest/openalex.py

```diff
@@ -102,3 +104,40 @@ def orcid_publications(orcid: str) -> Generator[dict, None, None]:
         time.sleep(1)
 
         yield from page
+
+
+def fill_in(snapshot: Snapshot, jsonl_file: Path) -> Path:
+    """Harvest OpenAlex data for DOIs from other publication sources."""
+    count = 0
+    with jsonl_file.open("a") as jsonl_output:
+        with get_session(snapshot.database_name).begin() as select_session:
+            stmt = (
+                select(Publication.doi)  # type: ignore
+                .where(Publication.doi.is_not(None))  # type: ignore
+                .where(Publication.openalex_json.is_(None))
+                .execution_options(yield_per=100)
+            )
+            for row in select_session.execute(stmt):
+                logging.info(f"filling in data for {row.doi}")
+                try:
```

Comment:
If we wanted to speed up the lookups I think we could get more than one DOI at a time?

https://github.com/sul-dlss/rialto-airflow/blob/94c1a28339a5255c14691312a0dd795b7b5909b3/rialto_airflow/harvest/openalex.py#L87

Reply:
Added a TODO comment about that.